src/ec/suite_b/ecdsa/verification.rs

```diff
@@ -194,7 +194,7 @@ fn twin_mul(
 ///
 /// See "`ECDSA_*_FIXED` Details" in `ring::signature`'s module-level
 /// documentation for more details.
-pub static ECDSA_P256_SHA256_FIXED: Algorithm = Algorithm {
+pub const ECDSA_P256_SHA256_FIXED: Algorithm = Algorithm {
```

Comment:
What I'd like to do:

1. Make this private static and rename: `static ECDSA_P256_SHA256_FIXED_VALUE: Algorithm = Algorithm { ... }`.
2. Expose a new `pub const`: `pub const ECDSA_P256_SHA256_FIXED: &'static signature::Algorithm = &ECDSA_P256_SHA256_FIXED_VALUE`.
3. Remove the `pub use suite_b::ecdsa::Algorithm as EcdsaVerificationAlgorithm`.

In other words, I am more interested in this change if it allows us to stop exporting `EcdsaVerificationAlgorithm` and related types. Does this work?

Reply:
Ah, nope, it doesn't work, for the same reason that you are filing this PR in the first place. It does work if the `_VALUE` part is `const` instead of `static` though. So, that's another advantage to doing this.

Reply:
Yup, you can have a const item be a `&'static` ref to another const item. So it'd be possible to expose only `&'static` things instead of proper `Algorithm`s. The downside, though, is that the types of all these top-level definitions would change, and you'd need a major version bump.

Reply:
The major version bump is not an issue. That's going to happen RSN for other reasons anyway.